### PR TITLE
Update latest numbers style

### LINKED ISSUE #759: Style new Latest Numbers to design spec
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the issue).**
- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).

See the new page created in #747 - style to the design in Figma (screenshot below)

Latest Numbers

### PR DESCRIPTION
**Pull request by theswerd**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [ ] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here:
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?

Made live updates page styling match figma

Closes #759 

## Did you add any dependencies?

No

## How did you test the change?

iOS simulator
Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-01 at 14 24 03

### CODE REVIEW

File: client/flutter/lib/pages/latest_numbers.dart [hunk 1]
```diff
@@ -73,17 +77,17 @@ class LatestNumbers extends StatelessWidget {
...
                     ),
                   ),
                   Container(
                     height: 25,
                   ),
                   Text(
-                    snapshot.hasData
-                        ? S.of(context).latestNumbersPageLastUpdated(lastUpd)
-                        : S.of(context).latestNumbersPageUpdating,
+                    snapshot.hasData ? 'Last updated $lastUpd' : S.of(context).latestNumbersPageUpdating,
```
Comment (advayDev1) [inline client/flutter/lib/pages/latest_numbers.dart:90]: missing loc function here too

**Comment by jamesblasco** [comment]
I think the card padding and the text style of the inner title 'GLOBAL CASES' are not accurate. This other pull request address this two problems better.
https://github.com/WorldHealthOrganization/app/pull/788

File: client/flutter/lib/pages/latest_numbers.dart (hunk 1 above)
Comment (advayDev1) [inline client/flutter/lib/pages/latest_numbers.dart:90]:
```suggestion
                    snapshot.hasData ? S.of(context).latestNumbersPageLastUpdated(lastUpd) : S.of(context).latestNumbersPageUpdating,
```

File: client/flutter/lib/pages/latest_numbers.dart [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import 'package:WHOFlutter/api/who_service.dart';
+import 'package:WHOFlutter/components/arrow_button.dart';
 import 'package:WHOFlutter/components/page_button.dart';
```
Comment (advayDev1) [inline client/flutter/lib/pages/latest_numbers.dart:3]:
```suggestion
```

**Pull request merged**